file-server/Dockerfile

```diff
@@ -1,3 +1,9 @@
+ARG TAG=dev
```

Comment:
Assuming we'll change this to default to `latest` once #282 is merged? Or do i have it backwards?

Reply:
Yeah probably.